R/coord-map.r

```diff
@@ -259,8 +260,8 @@ CoordMap <- ggproto("CoordMap", Coord,
     pos <- self$transform(x_intercept, panel_params)
 
     axes <- list(
-      bottom = guide_axis(pos$x, panel_params$x.labels, "bottom", theme),
-      top = guide_axis(pos$x, panel_params$x.labels, "top", theme)
+      top = guide_axis(pos$x, panel_params$x.labels, "top", theme),
+      bottom = guide_axis(pos$x, panel_params$x.labels, "bottom", theme)
```

Comment:
Just wondering: Is there anything happening here other than changing the order of the two lines? Is there a reason to switch the order?

Reply:
Yes, `scale_x$axis_order()` returns which axis is primary and which is secondary in this order (e.g. `"secondary" "primary"` meaning the top axis is secondary and the bottom is primary), so for the subsetting in the next line to work, this list needs to follow the same order (which is also why I had to reorder the values in `arrange <- panel_params$x.arrange %||% c("secondary", "primary")`)